_Tests/AudibleApi.Tests/EncryptionHelper.cs

```diff
@@ -0,0 +1,105 @@
+﻿using AuthorizationShared;
```

Comment:
I like this class. If you think it might have utility outside of testing, let's extract it to a more base library.

Reply:
I doubt it since it's pretty specific to Audible. We already effectively use the DecryptVoucher in AudibleDtos. We don't need EncryptVoucher because we aren't interested in serializing ContentLicense. We just need it for testing.